6_Model Comparison.ipynb

```diff
@@ -0,0 +1,6627 @@
+{
```

Comment:
<ul><li>`example notebook of PyMC3` =&gt; `example notebook from PyMC3`</li><li>I would add a little more discussion here about the general difficulties involved in model selection, particularly what is meant by "best" and "goodness". </li><li>`We try out two models` =&gt; `We try two models`</li></ul>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
- Changed

- I added the following paragraph:
By the 'goodness' of a model, we are referring to how compatible it is to the data. For example, if we have a linear and a quadratic model, we ask the question: Which model best fits the data? Note that here, we cannot use the likelihood to compare the two models. The likelihood simply increases as you increase the complexity of the model. So, we need to find a technique that compares the two models while also accounting for overfitting when using complex models.

- Changed